Wallet Download and Installation.md

```diff
@@ -0,0 +1,77 @@
+# Downloading and Installing Wallets to Your Devices
+
+Visit the **[Downloads](https://www.skycoin.net/downloads/)** page on skycoin.net to download the Skycoin Desktop Wallet and Skycoin Mobile Wallet for the device you are using:
+
+## Desktop_Wallet
+
+* ### Windows
+* ### macOS
+* ### Linux
+
+## Mobile_Wallet
+
+* ### iOS
+* ### Android
+
+## Desktop Wallet
+
+### Windows
+
+You can follow these steps to install the Skycoin Desktop Wallet on your PC:
+
+* Click on the download link next to "Windows Installer (Recommended)" to start downloading the Wallet installer.
+
+* Once the installer has been downloaded, double click to open it.
+
+* Confirm the action when prompted, after which the installation will be completed.
+
+* Once the installation is complete you can open the Wallet file from your Start Menu - Start -> Skycoin.
+
+### macOS
+
+You can follow these steps to install the Skycoin Desktop Wallet on your Mac:
+
+* Click on the download link next to "macOS Installer(Recommended)" to start downloading the .dmg file.
+
+* Once the .dmg file is downloaded, double click to open it.
+
+* A window will open where you can see the Desktop Wallet's icon and an arrow pointing towards the "Applications" folder. Drag and drop the Desktop Wallet's icon to the "Applications" folder.
+
+* Once this is done, you can now open the Desktop Wallet by double-clicking the Wallet from the "Applications" folder.
+
+### Linux
+
+You can follow these steps to install the Skycoin Desktop Wallet on your Linux:
+
+* Click on the download link next to "Linux Installer(recommended)" to start downloading the AppImage file.
+
+* Once the AppImage file is downloaded, right-click the file.
+
+* After right-clicking on the file, click on "Properties" section.
+
+* Click on the "Permissions" tab and select "Allow executing file as program".
+
+* Close the window and double click on the AppImage file, to install the Wallet.
+
+
+## Mobile Wallet
+
+### iOS
+
+You can follow these steps to install the Skycoin Mobile Wallet on your iPhone:
+
+* Click on the download link next to "Apple iOS App Store (recommended)" from your iPhone.
+
+* The link will open the App Store application on your Phone, clicking on "Get" will install the Wallet into your iPhone.
+
+* Once completed you can access the Skycoin Mobile Wallet from your Phone's menu.
+
+### Android
+
+You can follow these steps to install the Android version of the Skycoin Mobile Wallet:
+
+* Click on the download link next to "Google Play Store (recommended)" from your Android mobile.
+
+* The link will open your Playstore application, where you will find the button, "Install", pressing on which will begin the installation of the Wallet.
```

Comment:
Inconsistent naming/branding of Play Store. 

Also make two senteces out of this long one. where you will see an "Install" button. Press this to beging the installation of the wallet.